keystore/keystore.go

```diff
@@ -45,6 +48,14 @@ const (
 	scryptDKLen = 32
 )
 
+// prefixes for msg to be signed
+type PrefixType []byte
+
+var (
+	// PrefixRootUpdate is for signatures related to update the root of an identity
+	PrefixRootUpdate = []byte("rootupdate")
```

Comment:
Maybe "rootminorupdate"?

Reply:
right! done as `minorupdate`, as the concept only exists for roots (identity state)